unity/PingPongGameExample/PingPongGameSkeleton/Assets/Scripts/NetworkManager.cs

```diff
@@ -0,0 +1,342 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using System;
+using DistributedMatchEngine;
+using DistributedMatchEngine.PerformanceMetrics;
+using MobiledgeX;
+using System.Diagnostics;
+
+
+namespace MobiledgeXPingPongGame
+{
+    public class NetworkManager : MonoBehaviour
+    {
+
+#region Local Testing Variables
+
+        bool useAltServer = false; // set to true for local testing
+        string host = "localhost";
+        string altServerHost = "192.168.1.10"; // Local server hack. Override and set useAltServer=true for dev demo use.
+        int port = 3000;
+        string server = "";
+        
+#endregion
+       
+#region NetTest Variables
+
+        string l7Path; // Layer 7 in the OSI Networking Model (Application Layer) 
+        Stopwatch stopWatch = new Stopwatch();
+        NetTest netTest = null;
+
+#endregion
+
+        MobiledgeXIntegration integration;
+        public MobiledgeXWebSocketClient webSocketClient;
+        public GameSession gameSession;
+        public GameManager gameManager;
+        bool isPaused = false;
+        string queryParams = "";
+        string edgeCloudletStr = ""; // Connection url to Edge 
+
+
+#region MonoBehaviour Callbacks
+        // Use this for initialization
+        IEnumerator Start()
+        {
+            integration = new MobiledgeXIntegration();
+            // Demo mode DME server to run MobiledgeX APIs, or if SIM card is missing
+            // and a local DME cannot be located. Set to false if using a supported
+            // SIM Card
+            yield return  new Exception("RegisterClient NOT IMPLEMENTED");
```

Comment:
nit. extra space between "return" and "new"

Reply:
Also, maybe the exception should be "MobiledgeX Callflow NOT IMPLEMENTED" or something like that since you already clog "RegisterClient NOT IMPLEMENTED" in MobiledgeXAPICalls()